src/lib.rs

```diff
@@ -440,4 +440,78 @@ impl TxBuilder {
     }
 }
 
+struct BumpFeeTxBuilder {
+    txid: String,
+    fee_rate: f32,
+    rbf: Option<RbfValue>,
+    allow_shrinking: Option<String>,
+}
+
+impl BumpFeeTxBuilder {
+    fn new(txid: String, fee_rate: f32) -> Self {
+        Self {
+            txid,
+            fee_rate,
+            allow_shrinking: None,
+            rbf: None,
+        }
+    }
+
+    fn allow_shrinking(&self, address: String) -> Arc<Self> {
+        Arc::new(Self {
+            txid: self.txid.clone(),
+            fee_rate: self.fee_rate,
+            allow_shrinking: Some(address),
+            rbf: self.rbf.clone(),
+        })
+    }
+
+    fn enable_rbf(&self) -> Arc<Self> {
+        Arc::new(Self {
+            txid: self.txid.clone(),
+            fee_rate: self.fee_rate,
+            rbf: Some(RbfValue::Default),
+            allow_shrinking: self.allow_shrinking.clone(),
+        })
+    }
+
+    fn enable_rbf_with_sequence(&self, nsequence: u32) -> Arc<Self> {
+        Arc::new(Self {
+            txid: self.txid.clone(),
+            fee_rate: self.fee_rate,
+            rbf: Some(RbfValue::Value(nsequence)),
+            allow_shrinking: self.allow_shrinking.clone(),
+        })
+    }
+
+    fn build(&self, wallet: &Wallet) -> Result<Arc<PartiallySignedBitcoinTransaction>, Error> {
```

Comment:
The FFI layer uses `build` whereas the bdk library uses `finish`. Is there a reason for the difference in name? Wondering if we should fix that up before releasing 0.6.0.

Reply:
Yes good point, I think we should keep function names in sync where ever we can. Fixed.